GLTF/src/GLTFImage.cpp

```diff
@@ -23,6 +25,15 @@ GLTF::Image::Image(std::string uri, unsigned char* data, size_t byteLength, std:
 	}
 }
 
+GLTF::Image::~Image()
+{
+	const ImageCache::iterator it = std::find_if(_imageCache.begin(), _imageCache.end(), 
```

Comment:
Because we cache by uri, we should be able to do the lookup by the map key, no?

Reply:
Sorry abount the braces! We cache by full-path, but the URI passed to the `GLTFImage` constructor is just the filename of the full-path. I suggest to add a private constructor that remembers the key in the cache, whatever that key ends up to be in the future. That will majorly simplify the code. Is that okay for you?

Reply:
Yup, sounds good to me